### DISCUSSION TITLE
DBPool

### DISCUSSION THREAD

**Discussion by sysmat** [rbatis/rbatis#313] [General]
I'm new to this lib, is there any documentation/example how to:

1. how to configure connection **:** url, time out, validation, ssl
2. how to configure DB pool **:**  min, max connections, idle time, ...
3. how to Lazy init Rbatis with **:** DB pool

For MySQL data base

**Comment by zhuxiujia** [comment]
> I'm new to this lib, is there any documentation/example how to:
> 
> 1. how to configure connection **:** url, time out, validation, ssl
> 2. how to configure DB pool **:**  min, max connections, idle time, ...
> 3. how to Lazy init Rbatis with **:** DB pool
> 
> For MySQL data base

1.    
```toml  
rbdc = { version = "0.1",features = ["_tls-rustls"] }
```
2. rbatis use [deadpool](https://github.com/bikeshedder/deadpool)  so you can Dynamically adjust the number of connections.use 
```rust
 rb.get_pool().unwrap().resize(10)
```
3.  default is Lazy init. use rb.init() method

If you're still confused
see [document](https://rbatis.github.io/rbatis.io/#/)
or see [admin project](https://github.com/rbatis/abs_admin)
to Find specific ways to use it
